index.html

```diff
@@ -0,0 +1,160 @@
+            </p>
+        </div>
+
+        <hr class="main-separator">
+
+        <section class="personel-feature-container">
+            <div class="feature-container">
+                <h4>Language</h4>
+                <ul>
+                    <li>
+                        <p>JavaScript</p>
+                    </li>
+                    <li>
+                        <p>Ruby</p>
+                    </li>
+                    <li>
+                        <p>Html</p>
+                    </li>
+                    <li>
+                        <p>Css</p>
+                    </li>
+                </ul>
+            </div>
+            <div class="feature-container">
+                <h4>Frameworks</h4>
+                <ul>
+                    <li>
+                        <p>Bootstrap</p>
+                    </li>
+                    <li>
+                        <p>Ruby on Rails</p>
+                    </li>
+                    <li>
+                        <p>RSPec</p>
+                    </li>
+                    <li>
+                        <p>CapyBara</p>
+                    </li>
+                    <li>
+                        <p>Selenium</p>
+                    </li>
+                </ul>
+            </div>
+            <div class="feature-container">
+                <h4>Skills</h4>
+                <ul>
+                    <li>
+                        <p>Codekit</p>
+                    </li>
+                    <li>
+                        <p>Github</p>
+                    </li>
+                    <li>
+                        <p>Codepen</p>
+                    </li>
+                    <li>
+                        <p>Gitlab</p>
+                    </li>
+                    <li>
+                        <p>Terminal</p>
+                    </li>
+                </ul>
+            </div>
+        </section>
+
+        <hr class="main-separator feature-icon">
+
+        <section class="social-media-container">
+            <ul class="social-media">
+                <li>
+                    <a href="#">
+                        <img src="assets/github.png" alt="github.com logo">
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="assets/linkedin.png" alt="linkedin.com logo">
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <img src="assets/angellist.png" alt="angellist logo">
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <img src="assets/twitter.png" alt="Twitter Logo">
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <img src="assets/medium.png" alt="Medium.com logo">
+                    </a>
+                </li>
+            </ul>
```

Comment:
- [ ] Please make these social icons horizontal and remove `list-style` from them as the first reviewer mentioned and try to stick as much as possible with the given Figma design.

| Figma design | Your Design |
| ------------- | ------------- |
|  ![image](https://user-images.githubusercontent.com/68796232/159016853-1d8d9f76-32eb-448a-8d14-40fcd1367083.png) | ![image](https://user-images.githubusercontent.com/68796232/159016901-c614e301-9601-4886-9618-981937506366.png) |

Reply:
Please check the previous reviews from other reviewers. Every one of you has a different idea about the page size. On the design it is 375px.